app/Http/Controllers/Api/InventoryController1.php

```diff
@@ -0,0 +1,51 @@
+<?php namespace App\Http\Controllers;
+
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use App\Item;
+
+use Illuminate\Support\Facades\Response;
+use Illuminate\Support\Facades\Input;
+
+class InventoryController1 extends Controller {
+
+	/**
+	 * Display a listing of the resource.
+	 *
+	 * @param  string $query
+	 * @return json all or queried $items
+	 */
+	public function index($query)
+	{
+		$items = Item::all();
+
+		if(!is_null($query)) {
+			$items = $items->find($query);
+		}
+
+		if(is_null($items)) {
+			return Response::json("Not Found", 404);
```

Comment:
Think of this inventory endpoint as a collection. It always exists so it should never return 404. It's perfectly acceptable to return 200 and return JSON for an empty list. You probably don't even need a special check for that. Just `return $items;`.